packages/dashboard-frontend/src/containers/Loader/Workspace/Steps/StartWorkspace/index.tsx

```diff
@@ -171,7 +171,14 @@ class StepStartWorkspace extends AbstractLoaderStep<Props, State> {
         // do not switch to the next step
         return false;
       } catch (e) {
-        throw new Error(common.helpers.errors.getMessage(e));
+        const message = common.helpers.errors.getMessage(e);
+
+        if (common.helpers.errors.isError(e)) {
+          // throw original error
+          e.message = message;
```

Comment:
I don't think changing the original error message is a good idea.

```
throw new Error(message);
```
Is enough for each case

Reply:
I notice that if I throw a new error:
```
throw new Error(message);
```
, then this comparison [here](https://github.com/eclipse-che/che-dashboard/blob/53fc942af03fbbc11be8a08de5f1707cb8bfe162/packages/dashboard-frontend/src/pages/Loader/Workspace/index.tsx#L67) will always be false, because the original error (`RunningWorkspacesExceeded` error thrown [here](https://github.com/eclipse-che/che-dashboard/blob/53fc942af03fbbc11be8a08de5f1707cb8bfe162/packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts#L298)) is lost

In my new commit, I set it as:
https://github.com/eclipse-che/che-dashboard/blob/7aecab30a8fcf108bf959e9471c7860a0527a103/packages/dashboard-frontend/src/containers/Loader/Workspace/Steps/StartWorkspace/index.tsx#L175-L178

WDYT?

Reply:
@dkwon17  I think this problem should be solved there

https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts#L353-L363

```
        throw errorMessage;
```
->
```
        throw new Error(errorMessage);
```
WDYT?